lib/nib/debug.rb

```diff
@@ -21,7 +21,7 @@ def host
     @host = if ENV.key?('DOCKER_HOST_URL') && ENV['DOCKER_HOST_URL'] != ''
       URI.parse(ENV['DOCKER_HOST_URL']).host
     else
-      `ip route | head -n 1`.chomp
+      `ip route | awk 'NR==1 {print $3}'`.chomp
```

Comment:
This change reverted the change to this line made in #165. The flip-flop is due to the underlying OS change in the Docker image that's being used to run the test. In reality, this is brittle code deserves further consideration but is out of scope for this PR. Probably the best option is to remove this feature altogether, I'm not sure it's ever been userful.

Reply:
I am a little confused, what feature is this?